.config/protractor.conf.js

```diff
@@ -14,7 +14,10 @@ exports.config = {
     shardTestFiles: true,
     maxInstances: 4,
     chromeOptions: {
-      args: [ 'headless']
+      args: [ 'headless', '--window-size=1800x1080'],
```

Comment:
Are you sure that you need this specific size? maybe 1600*900 or 1920*1080 ?

Reply:
1600/900 or 1920/1080

Reply:
Done.